Tests/StreamChatTestTools/TestData/DummyData/ChannelPayload.swift

```diff
@@ -50,7 +50,7 @@ extension ChannelDetailPayload {
                     )
                 ],
                 createdAt: XCTestCase.channelCreatedDate,
-                updatedAt: .unique
+                updatedAt: XCTestCase.channelUpdateDate
```

Comment:
@evsaev This dummy is used in multiple places (30+). For now, I'll leave it like it is, to avoid a lot of changes in this PR. I just used the same pattern as before, change it in this PR is a bit out of scope